src/setup-java.ts

```diff
@@ -19,12 +20,52 @@ async function run() {
     const checkLatest = getBooleanInput(constants.INPUT_CHECK_LATEST, false);
     let toolchainIds = core.getMultilineInput(constants.INPUT_MVN_TOOLCHAIN_ID);
 
+    core.startGroup('Installed distributions');
+
     if (versions.length !== toolchainIds.length) {
       toolchainIds = [];
     }
 
-    core.startGroup('Installed distributions');
+    if (!versions.length) {
+      core.debug('JAVA_VERSION input is empty, looking for .java-version file');
+      const versionFileName = '.java-version';
```

Comment:
Have you considered adding the `java-version-file` input so that customers can specify file name and path to it?

Reply:
I don't think we need this input because `jenv` creates a file at the root of the project. Perhaps, if there are wishes from customers, we will be able to add this in the future, but for the first version, it seems to me that just adding this functionality is sufficient.